scipy/cluster/tests/test_vq.py

```diff
@@ -289,3 +289,25 @@ def test_kmeans_large_thres(self):
         res = kmeans(x, 1, thresh=1e16)
         assert_allclose(res[0], np.array([4.]))
         assert_allclose(res[1], 2.3999999999999999)
+        
+    def test_kmeans2_kpp_low_dim(self):
+        # Regression test for gh-11462
+        prev_res = np.array([[-1.95266667,  0.898],
+                             [-3.153375  ,  3.3945]])
+        np.random.seed(42)
+        res, _ = kmeans2(TESTDATA_2D, 2, minit='++')
+        assert_allclose(res, prev_res)
+
+    def test_kmeans2_kpp_high_dim(self):
+        # Regression test for gh-11462
+        n_dim = 100
+        size = 10
+        centers = np.vstack([ 5 * np.ones(n_dim),
+                             -5 * np.ones(n_dim)])
+        data = np.vstack([np.random.multivariate_normal(centers[0], np.eye(n_dim), size=size),
+                          np.random.multivariate_normal(centers[1], np.eye(n_dim), size=size)])
+        # 20x100 dataset
+        np.random.seed(42)
```

Comment:
just want to double-check--does it matter that this seeding happens after the random calls above?

Reply:
Thanks.

I fixed the styling.

The seed was mainly to fix the centers order because the k-means centers will almost surely be close to gaussian distribute mean. But it is better to be safe than sorry, I moved the seed position as recommended.